### ISSUE TITLE
Cygwin compile error

### ISSUE THREAD

**Issue by kishima** [mruby/mruby#4904]
I faced following compile error with latest mruby on cygwin environment. I don't see this error with mruby 2.0.0.

```
$ rake
CC    src/array.c -> build/host/src/array.o
gcc: エラー: C:cygwin64homekishimawork_mrubykishima_mrubysrcarray.c: No such file or directory
gcc: 致命的エラー: 入力ファイルがありません
コンパイルを停止しました。
gcc: エラー: C:cygwin64homekishimawork_mrubykishima_mrubysrcarray.c: No such file or directory
gcc: 致命的エラー: 入力ファイルがありません
コンパイルを停止しました。
rake aborted!
```

I think double quotes is missing for cygwin because of the commit b7bc03aa18f3cc5eac0bbd6690e24062df9fe837 .
If I add double quotes for filenames, then this error disappeared.

**Comment by matz** [comment]
We have merged #4905. Did the problem disappear?

**Comment by kishima** [comment]
I confirmed this issue was solved in cygwin environment. Thank you!
